src/PTY.ts

```diff
@@ -84,7 +102,8 @@ export async function linedOutputOf(command: string, args: string[], directory:
 }
 
 export async function executeCommandWithShellConfig(command: string): Promise<string[]> {
+    console.log('executeCommandWithShellConfig', command)
     const sourceCommands = (await loginShell.existingConfigFiles()).map(fileName => `source ${fileName} &> /dev/null`);
 
-    return await linedOutputOf(loginShell.executableName, ["-c", `'${[...sourceCommands, command].join("; ")}'`], process.env.HOME);
+    return await linedOutputOf(loginShell.executableName, ["/c", `"${[...sourceCommands, command].join(" && ")}"`], process.env.HOME);
```

Comment:
⚠ Warning: Unix-specific options were replaced by Windows-specific; need to find a portable way to express this option set.